src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -2038,4 +2038,16 @@ Additional information: {1}</value>
   <data name="AssemblyNameAndVersionDisplay" xml:space="preserve">
     <value>{0} ({1})</value>
   </data>
+  <data name="automatically_may_degrade_performance" xml:space="preserve">
+    <value>automatically (may degrade performance)</value>
```

Comment:
Most implicit features can degrade performance in some scenario. This wording sets a bad precedent.

```suggestion
    <value>automatically</value>
```

Reply:
I suppose i'd rather say: "will degrade performance" here :)

Reply:
Any suffix suggesting that the option will have a negative performance impact is in poor form here. If the goal is to make users aware of the performance cost of this option, it should be incorporated into Help &rarr; Performance Manager.